docs/ref/databases.txt

```diff
@@ -439,6 +439,24 @@ more details.
 .. _MySQL option file: https://dev.mysql.com/doc/refman/5.6/en/option-files.html
 .. _MySQLdb documentation: http://mysql-python.sourceforge.net/
 
+.. _mysql-sql-mode:
```

Comment:
Could you make this note a separate commit (perhaps resolving https://code.djangoproject.com/ticket/15940?) so we can backport it?